app/api/get-balance.ts

```diff
@@ -0,0 +1,4310 @@
+import axios from 'axios';
+import { TransactionResults } from '@blockstack/stacks-blockchain-sidecar-types';
+
+const api = 'https://sidecar.staging.blockstack.xyz/sidecar';
+
+export async function getAddressBalance(address: string) {
+  return axios.get(api + `/v1/address/${address}/balances`);
+}
+
+export async function getAddressTransactions(address: string) {
+  return axios.get<TransactionResults>(api + `/v1/address/${address}/transactions`);
+}
+
+const exampleTransactions = {
```

Comment:
LOL. Quite a stub file you have here. I like it.

Reply:
Thanks haha, just finishing up adding real txs to the store then I'm 👍